OpenLDAP/openldap/_meta/smart-descriptions.json

```
@@ -0,0 +1,41 @@
+[
+  {
+    "value": "Establish a connection from {source.ip} to {destination.ip}",
+    "conditions": [
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "destination.ip"
+      }
+    ]
```

I suggest adding a relationship:
```suggestion
      }
    ],
    "relationships": [
      {
        "source": "source.ip",
        "target": "destination.ip",
        "type": "connected to"
      }
    ]
```